test/python/test_onedim.py

```diff
@@ -991,11 +998,11 @@ def test_mixture_averaged(self, saveReference=False):
         if saveReference:
             np.savetxt(referenceFile, data, '%11.6e', ', ')
         else:
-            bad = utilities.compareProfiles(self.test_data_path / referenceFile, data,
+            bad = compareProfiles(test_data_path / referenceFile, data,
                                             rtol=1e-2, atol=1e-8, xtol=1e-2)
```

Comment:
The second line here should be reindented:
```suggestion
            bad = compareProfiles(test_data_path / referenceFile, data,
                                  rtol=1e-2, atol=1e-8, xtol=1e-2)
```
Also applies to other cases where `compareProfiles` is used.